src/ui/webeiddialog.cpp

```diff
@@ -355,6 +367,7 @@ bool WebEidDialog::event(QEvent* event)
 {
     if (event->type() == QEvent::LanguageChange) {
         ui->retranslateUi(this);
+        ui->versionLabel->setText(tr("Version: %1").arg(qApp->applicationVersion()));
```

Comment:
Why is this needed here in addition to the same line in constructor?

Reply:
retranslation might happen later or not at all